deploy/helm/falcosidekick/templates/deployment.yaml

```diff
@@ -18,8 +18,12 @@ spec:
       labels:
         app.kubernetes.io/name: {{ include "falcosidekick.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
+        aadpodidbinding: {{ include "falcosidekick.fullname" . }}
```

Comment:
Can you add a condition like `{{- if and .Values.config.azure.podIdentityClientID .Values.config.azure.podIdentityName -}}`